Libraries/Components/Touchable/TouchableWithoutFeedback.js

```diff
@@ -80,6 +80,14 @@ var TouchableWithoutFeedback = React.createClass({
      * is disabled. Ensure you pass in a constant to reduce memory allocations.
      */
     pressRetentionOffset: EdgeInsetsPropType,
+    /**
+     * This defines how far your touch can start away from the button. This is
+     * added to `pressRetentionOffset` when moving off of the button.
+     * The touch area never extends past the parent view bounds and the Z-index
```

Comment:
This should be more obvious to the reader, something like ** NOTE ** or even "Experimental" would help here